private-templates-service/client/src/components/AdaptiveCard/AdaptiveCard.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from "react";
+import * as AdaptiveCards from "adaptivecards";
+import { Card, Container, TemplateName } from "../AdaptiveCard/styled";
+
+function getCard() {
+	// Hard coded, will remove and connect to backend in future PR
+	var card = {
+		type: "AdaptiveCard",
+		version: "1.0",
+		body: [
+			{
+				type: "Image",
+				url: "http://adaptivecards.io/content/adaptive-card-50.png"
+			},
+			{
+				type: "TextBlock",
+				text: "Hello **Adaptive Cards!**"
+			}
+		],
+		actions: [
+			{
+				type: "Action.OpenUrl",
+				title: "Learn more",
+				url: "http://adaptivecards.io"
+			},
+			{
+				type: "Action.OpenUrl",
+				title: "GitHub",
+				url: "http://github.com/Microsoft/AdaptiveCards"
+			}
+		]
+	};
+	return card;
+}
+
+function GetRenderedCard() {
```

Comment:
I would split this function up into multiple smaller functions, for example:
- renderingSetup()
- parseCardTemplate() - you'd also need to handle an error case here - what if the parse function couldn't parse the data object you give it?
- renderAdaptiveCard()

Reply:
I wasn't too sure about what I was doing, but added some types. Also, for the function "renderAdaptiveCard()" on line 59, I didn't know how to create an HTMLElement so I left the return type as any. Would love some more feedback!

Reply:
You abstracted those into separate functions perfectly!! 👍 Much cleaner. And using 'any' is fine. (I had a quick search for a more appropriate HTML element type and couldn't find one either.)